### DISCUSSION TITLE
ArtStation.com

### DISCUSSION THREAD

**Discussion by Fawkes-Allen** [tophf/mpiv#61] [MPIV host rules] [answered]
I have been running into an issue trying to make a rule for Artstation or other sites like it that have an overlay that pops up over the image. I can't figure out how to properly get the image.

Random page for testing. https://www.artstation.com/wlop

**Comment by tophf** [comment, marked as answer]
```json
{"d": "artstation.com",
 "e": ".project-image *",
 "s": "return node.closest('a').querySelector('.image').src.replace(/(.+?\\/)(\\d{13,}\\/)?smaller_square(\\/.+)/, '$1large$3&dl=1')"
}
```

**Reply by Fawkes-Allen** [reply]
Thank you! Now I have an idea of how to fix a lot of the other sites I have been having issues with. And thank you so much for your work on this script.
